spec/Section 6 -- Execution.md

```diff
@@ -316,14 +316,28 @@ To execute a selection set, the object value being evaluated and the object type
 need to be known, as well as whether it must be executed serially, or may be
 executed in parallel.
 
+For the purposes of execution, the term "field" may be used to reference a
+"member" of a tagged type; this simplifies the wording of the algorithm.
```

Comment:
Thought for you: do we want to consider calling the tagged members "member fields"? By most accounts throughout the spec they operate like fields and perhaps it would remove the need for these reminders

Reply:
Agree; done.